packages/quantic/force-app/main/default/lwc/quanticFacetValue/quanticFacetValue.js

```diff
@@ -1,10 +1,47 @@
 import {LightningElement, api} from 'lwc';
+import LOCALE from '@salesforce/i18n/locale';
 
 export default class QuanticFacetValue extends LightningElement {
-  /** @type {import("coveo").FacetValue} */
   @api item;
   /** @type {boolean} */
   @api isChecked;
+  /** @type {string} */
+  @api variant = 'default';
+  /** @type {boolean} */
+  isDefaultFacet = false;
+  /** @type {boolean} */
+  isNumericFacet = false;
+  /** @type {boolean} */
+  isDateFacet = false;
```

Comment:
Suggestion. To really ensure there's no chance more than one of these are true at the same time.

```suggestion 
Variants = {
  Date: 'date',
  Numeric: 'numeric'
}

get isDefaultFacet() {
  return !this.variant;
}
 get isDateFacet() {
   return this.variant === Variants.Date;
 }
 get isNumericFacet() {
   return this.variant === Variants.Numeric;
 }
```

Reply:
more lines though. up to you

Reply:
agreed 👍